bigcache_test.go

```diff
@@ -34,7 +30,7 @@ func TestParallel(t *testing.T) {
 	go func() {
 		defer wg.Done()
 		for i := 0; i < keys; i++ {
-			sink, _ = cache.Get(fmt.Sprintf("key%d", i))
+			_, _ = cache.Get(fmt.Sprintf("key%d", i))
```

Comment:
@cristaloleg I removed `sink`. Why did we needed this?
BTW: In this test there are no assertion. Is this right?

Reply:
`sink` was made to ensure that compiler will not remove operation as no-op.

Reply:
How about deleteing this test? I think it duplicates `TestCacheDelRandomly`

https://github.com/allegro/bigcache/blob/ad8f119063878734a3466f293966e9da172e0213/bigcache_test.go#L438-L498